native/NativeSampleApps/test/RestExplorerTest/src/com/salesforce/samples/restexplorer/ServerPickerActivityTest.java

```diff
@@ -163,7 +163,6 @@ private void addCustomUrl(String label, String url) throws Throwable {
             openCustomEditDialog();
         }
         setText(com.salesforce.androidsdk.R.id.sf__picker_custom_label, label);
-        clickView(com.salesforce.androidsdk.R.id.sf__picker_custom_url);
```

Comment:
ARM emulator gets tripped up on the copy/paste menu when tapping the text.  Tapping on the text field isn't necessary for setting text.